app/views/users/index.html.erb

```diff
@@ -1 +1,8 @@
-<h1>index for users</h1>
\ No newline at end of file
+<% @users.each do |user|%>
+<div>
+  <img src='#{ <%= user.photo %> }' width='100' height='100' />
+  <span><%= user.name%></span>
+  <p>number of posts: <%=user.posts_counter || 0%></p>
+  <hr />
+</div>
```

Comment:
- [OPTIONAL ] Great job so far, But what do you think about using `Partials` which will allow you to extract pieces of code from your templates to separate files and also reuse them throughout your templates? 🤔 visit this link on [how to use Partials](https://riptutorial.com/ruby-on-rails/example/2875/partials) for more information 👍🏻 

Your view will look like: app/views/users/_user.html.erb, as a better practice instead of using @objects.each

```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

Reply:
i will consider that 
yeah, it's much better thank you